docs/podman.1.md

```diff
@@ -1,19 +1,28 @@
 % podman(1)
 
 ## NAME
-podman - Simple management tool for containers and images
+podman - Simple management tool for pods, containers and images
 
 ## SYNOPSIS
 **podman** [*options*] *command*
 
 ## DESCRIPTION
-podman is a simple client only tool to help with debugging issues when daemons
-such as CRI runtime and the kubelet are not responding or failing. A shared API
-layer could be created to share code between the daemon and podman. podman does not
-require any daemon running. podman utilizes the same underlying components that
-crio uses i.e. containers/image, container/storage, oci-runtime-tool/generate,
-runc or any other OCI compatible runtime. podman shares state with crio and so
-has the capability to debug pods/images created by crio.
+Podman is a fully featured container engine that is a simple client only tool.
+Podman is a Docker-compatible command line that eases the transition from other
+container engines and allows the management of pods, containers and images.
+Most Podman commands can be run as a regular user, without requiring additional
+privileges.  Best of all Podman accomplishes this without the use of a daemon.
```

Comment:
The last sentence seems redundant assuming daemonless is mentioned above.

Reply:
removed